planner_cspace/include/planner_cspace/blockmem_gridmap.h

```diff
@@ -213,7 +249,7 @@ class BlockMemGridmap : public BlockMemGridmapBase<T, DIM, NONCYCLIC>
       const BlockMemGridmap<T, DIM, NONCYCLIC, BLOCK_WIDTH, ENABLE_VALIDATION>& gm)
   {
     reset(gm.size_);
-    memcpy(c_.get(), gm.c_.get(), ser_size_);
+    memcpy(c_.get(), gm.c_.get(), ser_size_ * sizeof(T));
```

Comment:
This is a critical bug but it hasn't appeared as this method is called only when T is `char`.